process_report/process_report.py

```diff
@@ -302,6 +310,55 @@ def export_HU_only(dataframe, output_file):
     HU_projects.to_csv(output_file)
 
 
+def export_BU_only(dataframe: pandas.DataFrame, output_file):
+    def get_project(row):
+        project_alloc = row[PROJECT_FIELD]
+        if project_alloc.rfind("-") == -1:
+            return project_alloc
+        else:
+            return project_alloc[: project_alloc.rfind("-")]
+
+    BU_projects = dataframe[dataframe[INSTITUTION_FIELD] == "Boston University"]
+    BU_projects["Project"] = BU_projects.apply(get_project, axis=1)
+    BU_projects[SUBSIDY_FIELD] = 0
+    BU_projects = BU_projects[
+        [
+            INVOICE_DATE_FIELD,
+            PI_FIELD,
+            "Project",
+            COST_FIELD,
+            CREDIT_FIELD,
+            SUBSIDY_FIELD,
+            BALANCE_FIELD,
+        ]
+    ]
+    BU_projects = _apply_BU_subsidy(BU_projects)
+    BU_projects.to_csv(output_file)
+
+
+def _apply_BU_subsidy(dataframe):
+    subsidy_amount = 100
```

Comment:
@joachimweyl could the subsidy amount change in future? @QuanMPhm If it could change maybe this should be an argument that's passed down?

Reply:
I've added a cli argument for the user to specify the subsidy amount

Reply:
While the subsidy function is pretty general and it made sense to rename, this specific instance of the subsidy passed as argument is only applied to BU, so the CLI argument name for this specific subsidy should reflect that.